lunes_cms/cms/models/feedback.py

```diff
@@ -0,0 +1,34 @@
+# from django.contrib.contenttypes.fields import GenericForeignKey
+from django.contrib.contenttypes.models import ContentType
+from django.utils.translation import ugettext_lazy as _
+from django.db import models
+
+
+class Feedback(models.Model):
+    """
+    Contains a title, a description and a reference to the matter of feedback
+    """
+
+    title = models.CharField(max_length=255)
```

Comment:
```suggestion
```
I don't think we need a title... probably, users will only send one comment as feedback...